ChangeLog

```diff
@@ -91,6 +91,19 @@ Release date: TBA
   Insert your changelog randomly, it will reduce merge conflicts
   (Ie. not necessarily at the end)
 
+* ``missing-raises-doc`` will now check the class hierarchy of the raised exceptions
```

Comment:
Could you move these to just above L90? I can't make a suggestion and commit it myself as it is outside of the scope of the change 😢